changelog/added-big-endian-support-for-rtt.md

```diff
@@ -0,0 +1 @@
+Added big endian support for RTT
```

Comment:
I'm not sure if we need a third changelog entry for this PR. RTT either works with a target, or it doesn't, so I feel like this is covered by the ARMv7A changelog